src/feature-lifecycle/stabilization.md

```diff
@@ -1,22 +1,43 @@
 # Stabilizing features
 
-**Status:** Stub
+* **Status:** Current
+* **Last Updated:** 2022/03/10
```

Comment:
Let's use ISO's standard date format. Whenever I see a date in any other format I get paranoid the month and day might be swapped ^^'

```suggestion
* **Last Updated:** 2022-03-10
```